scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -614,6 +614,81 @@ private void setJobStatusToInErrorIfNotPaused(InternalJob job) {
         }
     }
 
+    TerminationData finishInErrorTask(JobId jobId, String taskName) throws UnknownTaskException, UnknownJobException {
+        JobData jobData = lockJob(jobId);
+        if (jobData == null) {
+            throw new UnknownJobException(jobId);
+        }
+        InternalJob job = jobData.job;
+        try {
+            InternalTask task = job.getTask(taskName);
+            if (task == null) {
+                throw new UnknownTaskException(taskName);
+            }
+            
+            TaskId taskId = task.getId();
```

Comment:
Here we can check not only if a task is alive but also if a task is In Error. 
I think it might be a good idea because otherwise one can terminate pending/skipped/faulty (all other tasks) than we we actually want. 
Testing it didn't show an immediate negative effect but we should prevent it.

One could write the following: 
TaskId taskId = task.getId();
 +            if (task.getStatus() != TaskStatus.IN_ERROR) {
 +                tlogger.info(task.getId(), "Task must be in state IN_ERROR: " + task.getStatus());
 +                return emptyResult(task.getId());
 +            }